### PR TITLE
chore: hot step test

### PR DESCRIPTION
**Pull request by LingyuCoder**
## Checklist

- [x] Tests updated (or not required).
- [x] Documentation updated (or not required).

### CODE REVIEW

**Comment by LingyuCoder** [comment]
!test-ng

File: .github/workflows/test-ng.yml [hunk 1]
```diff
@@ -57,6 +57,7 @@ jobs:
         run: pnpm run test:diff
 
       - name: Test new runner
+        continue-on-error: true
```
Comment (LingyuCoder) [inline .github/workflows/test-ng.yml:60]: !test-ng

File: .github/workflows/test-ng.yml [hunk 2]
```diff
@@ -58,7 +58,8 @@ jobs:
 
       - name: Test new runner
         run: |
-          OUTPUT=$(pnpm run test:ng --no-colors --silent=true 2>&1)
+          set -e;
+          OUTPUT=$(pnpm run test:ng --no-colors --silent=true 2>&1 || true)
```
Comment (LingyuCoder) [inline .github/workflows/test-ng.yml:62]: !test-ng

Comment (LingyuCoder) [reply .github/workflows/test-ng.yml:62]: !test-ng

Comment (LingyuCoder) [reply .github/workflows/test-ng.yml:62]: !test-ng

Comment (LingyuCoder) [reply .github/workflows/test-ng.yml:62]: !test-ng

File: .github/workflows/test-ng.yml [hunk 3]
```diff
@@ -58,7 +58,8 @@ jobs:
 
       - name: Test new runner
         run: |
-          OUTPUT=$(pnpm run test:ng --no-colors --silent=true 2>&1)
+          set -e;
```
Comment (LingyuCoder) [inline .github/workflows/test-ng.yml:61]: !test-ng

Comment (LingyuCoder) [reply .github/workflows/test-ng.yml:61]: !test-ng

**Review by hardfist — APPROVED** [review]

**Pull request merged**